pkg/interop/config.yaml

```diff
@@ -0,0 +1,14 @@
+join-servers:
```

Comment:
Shouldn't this be in `testdata`?

Reply:
shouldn't be checked in at all

Reply:
At least the file name is better than other files that you didn't check in 😉

Reply:
Good point :rofl: